test/fixtures/org/main.tf

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "test" {
+  source                    = "../../../1-org"
+  parent_folder             = var.parent_folder
+  org_id                    = var.org_id
+  billing_account           = var.billing_account
+  terraform_service_account = var.terraform_sa_email
+  default_region            = "us-east4"
```

Comment:
nit: it would be good to have some consistency with the regions used

Reply:
using `us-central1` for consistency with other fixtures